src/DynamoCoreWpf/Utilities/CrashUtilities.cs

```diff
@@ -16,14 +16,17 @@ internal static string GithubNewIssueUrlFromCrashContent(object crashContent)
             // provide fallback values for text content in case Resources or Assembly calls fail
             var issueTitle = Properties.Resources.CrashPromptGithubNewIssueTitle ?? "Crash report from Dynamo {0}";
             var dynamoVersion = AssemblyHelper.GetDynamoVersion().ToString() ?? "2.1.0+";
-            var content = GitHubCrashReportBody(crashContent);
```

Comment:
If this function `GitHubCrashReportBody` is no longer useful, we should remove it

Reply:
done